misc/python/materialize/cloudtest/k8s/cockroach.py

```diff
@@ -66,6 +68,9 @@ def __init__(self) -> None:
 
 
 class CockroachStatefulSet(K8sStatefulSet):
+    def __init__(self, namespace: str):
```

Comment:
Is this necessary, if the parent class also takes the namespace argument?

Reply:
No, you are right. I can remove it in this particular case.